.travis.yml

```diff
@@ -1,54 +1,21 @@
 language: node_js
 node_js:
-  - '12'
-  - '10'
-  - '8'
-  - '6'
+  # eslint 7: node 10+
+  # jest 26: node 10+
+  - 14
+  - 12
+  - 10
 install:
-  - 'yarn add "eslint@${ESLINT}" "jest@${JEST}" "babel-jest@${JEST}"'
-  - 'yarn'
+  - 'yarn add "eslint@${ESLINT}" "jest@${JEST}" "babel-jest@${JEST}" --ignore-engines'
```

Comment:
we shouldn't need `ignore-engines` anymore, but I guess it doesn't hurt either

Reply:
Removed 🙂